orion/sites/admin/src/announce/announce.styl

```diff
@@ -5,9 +5,48 @@
   height: 400px
   padding: 0 spacing2x
   box-sizing: border-box
-  color: colorBlackDark
+  color: colorGrayDark
+  font-family: fontFamilyRoboto /*not working*/
 
   & h1
     margin: spacing4x 0
     line-height: lineHeight3
     font-size: fontSize3
+  & ul
+    padding: 0
+
+  & .announce-lists
+    display: inline-block
+    list-style: none
+    padding-left: 0
+    height: lineHeight6
+    line-height: lineHeight6
+    border-bottom: 1px solid colorGrayLight
+
+    & li
+      display: inline-block
```

Comment:
you might not need this. There are no `li` elements inside `.announce-lists`.

Reply:
Oh wait, looks like the subheader has `li` elements while the main list has `div` elements. Can you consolidate to one or the other? Can you make both the subheader and main list have either `li` elements or `div` elements? Up to you which way you want to go.